cirq-core/cirq/sim/clifford/act_on_stabilizer_ch_form_args.py

```diff
@@ -12,34 +12,36 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-from typing import Any, Dict, TYPE_CHECKING, List, Sequence
+from typing import Any, Dict, List, Optional, Sequence
+from cirq.sim.clifford.clifford_simulator import CliffordState
 
 import numpy as np
 
+import cirq
 from cirq import value, ops, protocols
 from cirq.ops import common_gates, pauli_gates
 from cirq.ops import global_phase_op
 from cirq.sim.clifford.act_on_stabilizer_args import ActOnStabilizerArgs
 
-if TYPE_CHECKING:
-    import cirq
-
 
 class ActOnStabilizerCHFormArgs(ActOnStabilizerArgs):
     """Wrapper around a stabilizer state in CH form for the act_on protocol."""
 
     def __init__(
         self,
-        state: 'cirq.StabilizerStateChForm',
-        prng: np.random.RandomState,
-        log_of_measurement_results: Dict[str, Any],
-        qubits: Sequence['cirq.Qid'] = None,
+        state: Optional['cirq.StabilizerStateChForm'] = None,
+        initial_state: int = 0,
```

Comment:
Same, remove `initial_state` and change `state` to to `state: Union['cirq.StabilizerStateChForm', int] = 0`

Reply:
Thanks, you can resolve this one.